src/global.scss

```diff
@@ -9,3 +9,62 @@
 @import '~@ionic/angular/css/text-alignment.css';
 @import '~@ionic/angular/css/text-transformation.css';
 @import '~@ionic/angular/css/flex-utils.css';
+
+/** ---------------------Mat-form-field--------------------- */
```

Comment:
The styling wasn't working when put in `autocomplete-bar.component.scss` ?

Reply:
It probably would have worked, but these changes where made in global.scss so that they'll affect any future `mat-input` used within the app, even if it's not an autocomplete-bar component.